d2l-table.js

```diff
@@ -126,12 +126,19 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-table">
 				display: block;
 				width: 100%;
 				--d2l-table-border-color: var(--d2l-color-mica);
+				--d2l-table-border: 1px solid var(--d2l-table-border-color);
 				--d2l-table-header-background-color: var(--d2l-color-regolith);
-				--d2l-table-border-overflow: dashed 1px var(--d2l-color-mica);
+				--d2l-table-border-radius: 0.3rem;
+				--d2l-table-row-border-color-selected: var(--d2l-color-celestine);
+				--d2l-table-row-background-color-selected: var(--d2l-color-celestine-plus-2);
```

Comment:
Merged these with what was in `d2l-table-shared-styles.js` and are defining them on the element instead of globally -- this matches how we do things in Lit as well.